### PR TITLE
Swerve velocity control testing

### PR DESCRIPTION
**Pull request by Tan560**
I initially had only it using two slot, like shown below. I think it is unnecessary, since we control the gains via XML and use the m_useFOC there , but I can change it back if you think we need it. 
        if (m_useFOC)
        {
            configs.Slot1.kS = m_driveKs; // To account for friction, static feedforward
            configs.Slot1.kV = 0;         // Torque-based velocity does not require a feed forward, as torque will accelerate the rotor up to the desired velocity by itself
            configs.Slot1.kP = m_driveKp;
            configs.Slot1.kI = m_driveKi;
            configs.Slot1.kD = m_driveKd;
        }
        else
        {
            configs.Slot0.kS = m_driveKs; // To account for friction, static feedforward
            configs.Slot0.kV = m_driveKf; // Kraken X60 is a 500 kV motor, 500 rpm per V = 8.333 rps per V, 1/8.33 = 0.12 volts / rotation per second(Free wheeling)
            configs.Slot0.kP = m_driveKp;
            configs.Slot0.kI = m_driveKi;
            configs.Slot0.kD = m_driveKd;
        }

### CODE REVIEW

File: src/main/deploy/chassis/swervemodule_9999.xml [hunk 1]
```diff
@@ -1,7 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <swervemodule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <attributes
-    proportional="75.0"
-    integral="55.0"
-    derivative="0.0" />
+    turn_proportional="75.0"
```
Comment (witcpalek) [inline src/main/deploy/chassis/swervemodule_9999.xml:4]: Shouldn't this file go away since you deleted the practice bot?

Comment (Tan560) [reply src/main/deploy/chassis/swervemodule_9999.xml:4]: Yes it should, I forgot that this was the practice bot. I will delete it now

**Review by witcpalek — APPROVED** [review]

File: src/main/cpp/chassis/headingStates/MaintainHeading.cpp [hunk 2]
```diff
@@ -49,7 +49,7 @@ void MaintainHeading::UpdateChassisSpeeds(ChassisMovement &chassisMovement)
 
             auto radianCorrection = m_controller.Calculate(currentAngle.value(), targetAngle.value());
 
-            correction = abs(radianCorrection) > 0.2 ? units::angular_velocity::radians_per_second_t(radianCorrection) : units::angular_velocity::radians_per_second_t(0.0);
+            correction = abs(radianCorrection) > 0.1 ? units::angular_velocity::radians_per_second_t(radianCorrection) : units::angular_velocity::radians_per_second_t(0.0);
```
Comment (witcpalek) [inline src/main/cpp/chassis/headingStates/MaintainHeading.cpp:52]: get rid of magic number

File: src/main/deploy/chassis/swervemodule_9997.xml [hunk 3]
```diff
@@ -2,6 +2,8 @@
 <swervemodule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <!--- TO BE TUNED -->
   <attributes
+    useFOC="false"
```
Comment (witcpalek) [inline src/main/deploy/chassis/swervemodule_9997.xml:5]: make this true

**Review by witcpalek — CHANGES_REQUESTED** [review]

**Review by witcpalek — APPROVED** [review]

**Pull request merged**